tests/scripts/myapp.py

```diff
@@ -33,7 +28,7 @@ def train(d: str):
     w2 = torch.randn(H, D_out, device=device)
 
     learning_rate = 1e-6
-    for t in tqdm(range(1000)):
+    for t in range(5):
```

Comment:
Is this enough time to actually log anything, i think we need at least 10 seconds for the first log, since we're polling